lib/datadog/core/telemetry/event.rb

```diff
@@ -242,15 +245,30 @@ def initialize(changes, origin)
           end
 
           def payload(seq_id)
-            {
-              configuration: @changes.map do |name, value|
-                {
-                  name: name,
-                  value: value,
-                  origin: @origin,
-                }
-              end
-            }
+            { configuration: configuration(seq_id) }
+          end
+
+          def configuration(seq_id)
+            config = Datadog.configuration
+
+            res = @changes.map do |name, value|
+              {
+                name: name,
+                value: value,
+                origin: @origin,
+              }
+            end
+
+            unless config.dig('appsec', 'sca_enabled').nil?
+              res << {
+                name: 'appsec.sca_enabled',
+                value: config.appsec.sca_enabled,
+                origin: 'code',
+                seq_id: seq_id,
+              }
+            end
+
+            res
```

Comment:
This payload is only used for remote configuration changes today.
Is `DD_APPSEC_SCA_ENABLED` remotely configurable?
I asked because this method you have no product specific code, everything should come through the `changes` argument to `AppClientConfigurationChange.new(changes, origin)`.

Reply:
According to this document (https://docs.google.com/document/d/12NBx-nD-IoQEMiCRnJXneq4Be7cbtSc6pJLOFUWTpNE), DD_APPSEC_SCA_ENABLED is remotely configurable